embeddings/full_post.md

```diff
@@ -58,42 +58,42 @@ Facebook AI Research's FastText treats words as n-grams (subwords) of characters
 * **Better at Representing Rare Words:** By breaking down words into manageable parts, FastText improves the embeddings of less common vocabulary.  
   
 ### 1.5 Contextual Embeddings  
-------------------------------  
+---  
   
-#### ELMo (Embeddings from Language Models)  
-------------------------------------------  
+#### ELMo (Embeddings from Language Models)  (2018)
+---  
   
 * **Context-Dependent Word Representations:** Provides dynamic embeddings based on the word's context within a sentence.  
   
-#### BERT (Bidirectional Encoder Representations from Transformers)  
-----------------------------------------------------------------  
+#### BERT (Bidirectional Encoder Representations from Transformers) (2018)
+---  
   
 * **Transformer Architecture:** Utilizes the powerful transformer model.  
 * **Pre-Training Tasks:** Masked language modeling and next sentence prediction enable deep contextual understanding.  
   
 #### GPT Series  
-----------------  
+---  
   
 * **Developed by OpenAI:** Focuses on generative tasks with a series of transformer-based architectures.  
 * **Generative Capabilities:** Excels in tasks requiring the generation of coherent text.  
   
 ### 1.6 Recent Advances  
--------------------------  
+---  
   
 #### Transformer Models  
------------------------  
+---  
   
-* **Improved Parallelization:** Enables faster training through parallelizable architecture.  
+* **Improved Parallelization:** Enables faster training through parallelizable architecture. [Sugguested Link: Attention is All You Need](https://arxiv.org/abs/1706.03762)  
```

Comment:
"Suggested". Nit: I would put "Suggested Link: " outside the `[...]`, but up to you 😁